Assignment3/main.js

```diff
@@ -0,0 +1,31 @@
+var tools = require("./problem1.js");
+var array = [1, 7, 10, 3, 8];
+var value = tools.secondLargest(array);
+console.log("Program 1 --------");
+console.log("Array = ["+array+"]");
+console.log("Second Largest Value: "+value);
+console.log('\n');
+
+var tools = require("./problem2.js");
+var string = "aarushi";
+var value = tools.calculateFrequency(string);
+console.log("Program 2 --------");
+console.log("String = "+string);
+console.log("Frequency values are ");
+console.log(value);
+console.log('\n');
+
+var tools = require("./problem3.js");
+var input = { "flatJSON": false, "i": { "am": { "not": { "so": { "flat": true, "unflat": false } }, "a": "tree" } }, "dates": [ { "day": 1 }, { "day": 8947 } ] }
```

Comment:
a JS file should have atmost 100 characters in one line. Break the object declaration to multi-line.